tests/test-cluster.py

```diff
@@ -87,7 +98,9 @@ def _setup_lxc(self, channel_or_snap, launch_configs=None):
                 ).split()
             )
             time.sleep(20)
+
```

Comment:
Do we need these empty lines?

Reply:
Makes the separation b/w different steps easily visible imo.